core/src/main/java/fr/sncf/osrd/api/InfraCacheStatusEndpoint.java

```diff
@@ -44,15 +44,18 @@ public Response act(Request req) throws IOException {
         // Parse request input
         try {
             var body = new RqPrint(req).printBody();
-
-            if (body.equals("")) {
+            String infra;
+            if (!body.equals("")) {
+                infra = adapterRequest.fromJson(body).infra;
+                if (infra != null) {
+                    var request = adapterRequest.fromJson(body);
```

Comment:
Can't we parse that once, then check is `request.infra` is null?

Additionally, we should probably return an error if it is.